Dell/recovery_common.py

```diff
@@ -655,12 +691,11 @@ def parse(fname):
                     arch, package, modaliases = parse(os.path.join(repo, fname))
                     if not modaliases and (arch == "all" or arch == my_arch):
                         to_install.append(package)
-
     return to_install
 
 def create_grub_entries(target_dir='/target', rec_type='hdd'):
     '''Create GRUB entry for dell-recovery during ubiquity installation'''
-    env = os.environ
+    #env = os.environ
```

Comment:
if i'm not mistaken, it was important to have the environment set here so that the language would be applied as necessary for any translations that do exist.

Reply:
Yes, don't need to change.
Thanks.